### PR TITLE
ISSUE-2669:Upgrade Netty to 4.1.60.final

### PR DESCRIPTION
**Pull request by ruanwenjun**
### Motivation

see #2669 

### Changes

upgrade the netty version to 4.1.60.final

Master Issue: #2669

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -866,7 +866,6 @@
           <source>${javac.target}</source>
           <target>${javac.target}</target>
           <compilerArgs>
-            <compilerArg>-Werror</compilerArg>
```
Comment (dlg99) [inline pom.xml:869]: why do you need this change?  This is a build-wide setting.

If netty upgrade resulted in some deprecation warnings you need to fix them

**Review by dlg99 — CHANGES_REQUESTED** [review]
Please undo the werror change, fix anything that blocks the build after that, and update the license files.

File: pom.xml (hunk 1 above)
Comment (ruanwenjun) [reply pom.xml]: Yeal, I try to restore this configuration and update the dependency license files. And now I get a problem about `TestTLS`.

**Comment by dlg99** [comment]
Netty is upgraded to 4.1.63 in https://github.com/apache/bookkeeper/pull/2696 ; I'll close this one

**Pull request closed without merging**